### DISCUSSION TITLE
Watsonx (IBM) support?

### DISCUSSION THREAD

**Discussion by carlbberg** [FlowiseAI/Flowise#884] [Ideas]
Hi folks, any plans to support additional LLM providers such as watsonx (https://www.ibm.com/products/watsonx-ai) in addition to OpenAI, Huggingface, Cohere, Azure OpenAI, etc? Those enterprise tools will become more prevalent and popular moving forward, it would be nice to keep an eye on that.

Thanks

**Comment by chungyau97** [comment]
Hi @carlbberg, 

I believe it's not yet implemented in LangChain JS

Reference:
[LangChain JS Language models](https://js.langchain.com/docs/modules/model_io/models/)

**Comment by denchanca** [comment]
any movement on this?

**Comment by HenryHengZJ** [comment]
supported now!
